README.md

```diff
@@ -1,7 +1,5 @@
 # chia-operator
 
-> NOTE: This is an early implementation that is being actively worked on. The API should stay backwards-compatible in the majority of cases but is not currently guaranteed.
```

Comment:
This note's been up long enough, and so many changes have been made to the operator since then. Probably can't commit to nothing ever changing in the current API, but it should be scarce enough that I'm fine with getting rid of this note.